util/util.go

```diff
@@ -58,3 +63,50 @@ func AddImageNames(store storage.Store, image *storage.Image, addNames []string)
 	}
 	return nil
 }
+
+func ContextFromGlobalOptions(c *cli.Context, flagPrefix string) (*types.SystemContext, error) {
+	ctx := &types.SystemContext{
+		RegistriesDirPath: c.GlobalString("registries.d"),
```

Comment:
I wasn't planning to add these at this time.  Was on the fence about ripping them out or not and kept them as they were in skopeo.  They're currently all no-ops in Buildah.  In skopeo  RegistriesDirPath is used for lookaside Docker signature storage and is a build time option.  I'm thinking we problem don't need this.  I'll try removing.